webhooks-extension/README.md

```diff
@@ -50,6 +56,10 @@ curl -d "${data}" -H "Content-Type: application/json" -X POST http://localhost:9
 
 Reference the [Knative eventing GitHub source sample](https://knative.dev/docs/eventing/samples/github-source/) to properly create the `accesstoken` secret. This is the secret that is used to create GitHub webhooks.
 
+If using Helm, you can also specify a Helm release name that enables you to do `helm install --upgrade` as expected, otherwise our sink will include the short Git commit ID in your Helm release name.
```

Comment:
Can you also please add a link to the helm pipeline that people can use with this extension. This could go in the [Limitations](https://github.com/tektoncd/experimental/tree/master/webhooks-extension#limitations) section with the [simple-pipeline](https://github.com/pipeline-hotel/example-pipelines/blob/master/config/pipeline.yaml) link?